ml-agents-envs/mlagents_envs/rpc_communicator.py

```diff
@@ -56,9 +57,7 @@ def create_server(self):
 
         try:
             # Establish communication grpc
-            self.server = grpc.server(
-                ThreadPoolExecutor(max_workers=10), options=(("grpc.so_reuseport", 0),)
```

Comment:
I assume this has something to do with this GRPC issue? https://github.com/grpc/grpc/issues/18210

Seems like it may be resolving itself in the latest `grpcio` package versions.  In any case, do you recall why we used `grpc.so_reuseport` to begin with?  I believe it's a performance optimization, so wondering if we lose anything.

Reply:
I reverted that change. It was a leftover of an experiment I was making.